otherlibs/dune-rpc/private/exported_types.mli

```diff
@@ -138,6 +138,7 @@ module Job : sig
     val hash : t -> int
     val create : int -> t
     val sexp : (t, Conv.values) Conv.t
+    val to_string : t -> string
```

Comment:
Could you use the sexp converter insead of adding this function?

Reply:
Done.